.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 ---
+exclude: '(esmvaltool/diag_scripts/extreme_events/climdex.pcic.ncdf|esmvaltool/diag_scripts/cvdp)'
```

Comment:
Would it be possible to make this multiple lines? with one path per line?

Reply:
Yeah I was thinking about this too, but I'm not quite sure how to do multi-line yaml for regex.

Reply:
Maybe you could use what is described [here](https://yaml-multiline.info/) under Flow scalars?

Reply:
Oof, this was more tricky than expected. Apparently you need the `(?x)` before. I'm not entirely what it does, but all pre-commit configs with multi-line regexes that I checked have it. And it seems to work this way.